src/utils/generateJailsKeyInsightsData.js

```diff
@@ -24,11 +24,12 @@ import {
 import months from "../constants/months";
 import formatNumber from "./formatNumber";
 import getLastUpdatedDate from "./getLastUpdatedDate";
+import isDifferentPercentageCovered from "./isDifferentPercentageCovered";
 
 const generateCountiesCaption = (countyCoverage, populationCoverage) => {
   return `Currently, about ${formatNumber(
     countyCoverage
-  )} percent of counties report their jail populations on at least a monthly basis, representing about ${formatNumber(
```

Comment:
nit: this can stay "jail populations". We only collect jail populations from counties, and then we calculate the incarceration rate from it.